ibis/backends/pandas/execution/window.py

```diff
@@ -210,16 +216,20 @@ def trim_window_series(data, timecontext: Optional[TimeContext]):
     if time_col not in df:
```

Comment:
Can we raise error here? I believe this is why we didn't catch this for grouped case.